### DISCUSSION TITLE
Figtree as app font?

### DISCUSSION THREAD

**Discussion by inari-codes** [imashnake0/Animite#129] [General]
https://fonts.google.com/specimen/Figtree

Just found this font on Google Fonts and I think it's kind of a fit for Animite and MD3 since it's kind of soft while still being general similar to Google Sans Text, just a suggestion

**Comment by inari-codes** [comment]
![Screenshot_20240315-212603_Read You](https://github.com/imashnake0/Animite/assets/85934291/eeb485a5-0f07-4eb0-b901-17eb989aeea4)
Some demos
![Screenshot_20240315-212305_Read You](https://github.com/imashnake0/Animite/assets/85934291/4d235038-555c-4cd0-92c5-a84b3aa4f7f1)
![Screenshot_20240315-212413_Read You](https://github.com/imashnake0/Animite/assets/85934291/447884a3-dd22-41b7-84f5-91c469c954bf)

**Comment by tanmaytez** [comment]
This app has the potential to be one of the good anilist clients. Please provide more functionality in the upcoming updates

**Reply by boswelja** [reply]
This app is open source and accepting contributions. Please contribute anything you'd like to see, or at the very least open a discussion instead of hijacking a font discussion 🙂
